lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -366,8 +369,33 @@
 .pagination-buttons {
     display: flex;
     padding: 2rem;
+    align-items: center;
     justify-content: flex-end;
     height: fit-content;
+
+    button {
+        color: $color-bg-blue-dark;
+        height: 44px;
+        width: 44px;
+        margin-bottom: 0;
+        margin-left: 10px;
+        text-align: center;
+
+        p {
+            margin-bottom: 0;
+        }
+    }
+
+    > div {
+        background-color: $color-bg-white;
+        color: $color-bg-blue-dark;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 44px;
+        width: 44px;
+        text-align: center;
+    }
```

Comment:
This is not needed if we don't have the ellipsis.

Reply:
Ok, unnecessary styling is now removed. Background color and the left margin for the pagination buttons have been kept to position and add color to the button text.